result/flow.yaml

```diff
@@ -15,12 +15,14 @@ components:
           x-constraint:
           - "/components/schemas/Flow/properties/name"
           x-field-uid: 1
-        metric_groups:
+        metric_tags:
           description: >-
-            A list of metric groups used to disaggregate flows.
-            A metric group that does not exist for a flow group MUST return
+            A list of metric tags used to disaggregate flows.
```

Comment:
What will be the expected behavior if "metric tags" list is empty?

Reply:
I think we should specify that the disaggregation would apply only for the specified "metric tags"; all configured tracking that is not specified in result, stats will be consolidated for them. So by default, empty "metric tags" will have no disaggregated stats.